usaspending_api/search/v2/views/search.py

```diff
@@ -182,17 +186,17 @@ def post(self, request):
                     .filter(awarding_toptier_agency_name__isnull=False) \
                     .values(
                         agency_name=F('awarding_toptier_agency_name'),
-                        agency_abbreviation=F('awarding_toptier_agency_abbreviation')) \
-                    .annotate(aggregated_amount=Sum('federal_action_obligation')) \
+                        agency_abbreviation=F('awarding_toptier_agency_abbreviation'))
+                queryset = sum_transaction_amount(queryset, 'aggregated_amount', filter_types=filter_types) \
```

Comment:
if this is being called on every queryset for the different filter types, should it be applied at once, maybe at the end, to reduce duplication?